README.md

```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# CSS Homework1
+
+> ## Fernando Alonso Regalado
+
+![Bio Picture](https://github.com/fernandoregalado/hw01/blob/master/picture_bio.jpg)
+
+Hi! I am a second year **MPP student** :neckbeard: and peruvian **economic analyst** :bowtie:.
+
+* :school_satchel: Education: [*Harris School of Public Policy*](https://harris.uchicago.edu/)
```

Comment:
Maybe you don't need to make Harris School of Public Policy as well as Central Reserve Bank of Peru Italic. For me, keeping it in the original format looks better.